src/containers/Historical/Historical.js

```diff
@@ -1,13 +1,78 @@
-import React from 'react'
+import React, { useState } from 'react'
 import './Historical.css'
-// import { connect } from 'react-redux'
-// import * as actions from '../../actions'
+import { connect } from 'react-redux'
+import * as actions from '../../actions'
 import { Link } from 'react-router-dom'
 
-const Historical = () => {
+const Historical = ({ userProfile, userQuote, loadSolarData }) => {
+  const [january, updateJanuary] = useState(undefined)
+  const [february, updateFebruary] = useState(undefined)
+  const [march, updateMarch] = useState(undefined)
+  const [april, updateApril] = useState(undefined)
+  const [may, updateMay] = useState(undefined)
+  const [june, updateJune] = useState(undefined)
+  const [july, updateJuly] = useState(undefined)
+  const [august, updateAugust] = useState(undefined)
+  const [september, updateSeptember] = useState(undefined)
+  const [october, updateOctober] = useState(undefined)
+  const [november, updateNovember] = useState(undefined)
+  const [december, updateDecember] = useState(undefined)
 
```

Comment:
We might come back to this and try to implement a custom hook for all of the historical data inputs